charts/Makefile

```diff
@@ -1,6 +1,8 @@
 ROOT_DIR := ..
 include $(ROOT_DIR)/Makefile.env
 
+WITHOUT_OPENSHIFT ?= true
```

Comment:
Why without? Make it positive. It will also fix the wrong usage bellow :)

Reply:
Other places in the code such as third_party/egeria/deploy.sh uses WITHOUT_OPENSHIFT ... should I leave it afterall to make it consistent? tnx

Reply:
You pass it as is to `global.openshift` so usage is wrong here (installing with openshift=true when WITHOUT_OPENSHIFT=true.

As far as I can tell WITHOUT_OPENSHIFT is only used in the old vault deploy script and in Egeria deploy script. I think it would be better to change it in the other places instead but whatever you want.

Reply:
OK, I'll change it then (and fix the usage) tnx